mmv1/third_party/terraform/.teamcity/components/builds/build_triggers.kt

```diff
@@ -18,10 +18,20 @@ import jetbrains.buildServer.configs.kotlin.triggers.schedule
 class NightlyTriggerConfiguration(
```

Comment:
I assume that all the changes in this PR need to be synced to the 6.0.0 release branch first before it can be used for release branch testing?

Reply:
I don't think so - this PR would be merged into main and TeamCity will use the configuration present on main branch of the TPG repo to control what's in TeamCity.

I believe the only coordination required for these types of changes is that the [feature branch(es) being referenced](https://github.com/GoogleCloudPlatform/magic-modules/blob/ef55f56f8759510005768fda6bf22a04d44b1b9f/mmv1/third_party/terraform/.teamcity/components/projects/feature_branches/FEATURE-BRANCH-major-release-6.0.0.kt#L18) would need to exist before we merge changes to the config.

Reply:
Nice! Thanks for the explanation. I must be confused with some testing step we did in the past that we needed to have the configuration checked into the branches.